nixos/modules/system/boot/kernel.nix

```diff
@@ -156,6 +156,16 @@ in
       description = "List of modules that are always loaded by the initrd.";
     };
 
+    boot.initrd.includeDefaultModules = mkOption {
+      type = types.bool;
+      default = true;
+      description = ''
+        This option, if set, adds a collection of default kernel modules
+        to `<option>boot.initrd.availableKernelModules</option>` and
```

Comment:
Just a nitpick (sorry): can you remove the backticks? `<option>` already makes the text visually distinct.

Reply:
Done :+1: